cms/private_media/views.py

```diff
@@ -0,0 +1,34 @@
+from typing import TYPE_CHECKING
+
+from django.core.exceptions import PermissionDenied
+from django.shortcuts import redirect
+from wagtail.images.permissions import permission_policy
+from wagtail.images.views.serve import ServeView
+
+if TYPE_CHECKING:
+    from django.http import HttpResponse
+
+    from cms.private_media.models import AbstractPrivateRendition, PrivateImageMixin
+
+
+class PrivateImageServeView(ServeView):
+    def serve(self, rendition: "AbstractPrivateRendition") -> "HttpResponse":
+        image: PrivateImageMixin = rendition.image
```

Comment:
Nit: This might as well be typed as the image model itself

Reply:
I am pretty sure pylint then complains that `is_public` and `file_permissions_are_outdated` are not valid attributes

Reply:
If that's the case, we should probably disable `pylint`'s attribute checking, and instead let `mypy` do it (which is probably better at it anyway).

Reply:
Is is a big deal? This solves the issue whilst remaining concise

Reply:
@RealOrangeOne Could you mark this one as resolved please?